x-pack/plugins/ui_actions_enhanced/public/components/action_wizard/action_wizard.test.tsx

```diff
@@ -80,7 +80,7 @@ test('If not enough license, button is disabled', () => {
   // check that all factories are displayed to pick
   expect(screen.getAllByTestId(new RegExp(TEST_SUBJ_ACTION_FACTORY_ITEM))).toHaveLength(2);
 
-  expect(screen.getByText(/Go to URL/i)).toBeDisabled();
```

Comment:
There is button with a lot of nested content inside. It stopped returning a button, but returns a nested `p` instead.